mods/ts/sequences/vehicles.yaml

```diff
@@ -40,6 +41,8 @@ hvr:
 	Inherits: ^VehicleOverlays
 	muzzle: gunfire
 		Length: *
+		AlphaFade: True
```

Comment:
In my opinion `AlphaFade` isn't suitable for `gunfire.shp`, it is only 3 frames long and this nearly cancels out `IgnoreWorldTint` in terms of brightness and makes the last frame practically invisible.

In my opinion we should either go with a mellow custom Alpha scaling like
`Alpha: 1, 0.8, 0.6`,
or just remove the fading completely.

Reply:
I removed it.